### PR TITLE
gh-3011 Rebranding issues

### PR DESCRIPTION
**Pull request by elgatovital**
Handles #3011

Changes proposed in this pull request:
- Adjust strings

### CODE REVIEW

File: Multisig/UI/App/Onboarding/OnboardingViewController.swift [hunk 1]
```diff
@@ -51,8 +51,8 @@ class OnboardingViewController: UIViewController {
         super.viewDidLoad()
 
         completelyNewLabel.setStyle(.callout)
-        loadSafeButton.setText("Load existing Safe", .bordered)
-        createSafeButton.setText("Create new Safe", .filled)
+        loadSafeButton.setText("Load existing Safe Account", .bordered)
```
Comment (liliya-soroka) [inline Multisig/UI/App/Onboarding/OnboardingViewController.swift:54]: could you check if "Load existing Safe Account" is not too long for the button? I would leave "Load existing Account" . Same for the create button

File: Multisig/UI/Safe Management/Add Safe/CreatePasscodeSuggestionViewController.xib [hunk 2]
```diff
@@ -41,8 +41,8 @@
                             <nil key="highlightedColor"/>
                         </label>
                         <label opaque="NO" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="251" verticalHuggingPriority="251" textAlignment="center" lineBreakMode="tailTruncation" numberOfLines="0" baselineAdjustment="alignBaselines" adjustsFontSizeToFit="NO" translatesAutoresizingMaskIntoConstraints="NO" id="MkS-TR-Xqe">
-                            <rect key="frame" x="0.0" y="38.5" width="374" height="61"/>
-                            <string key="text">Protect your data with a passcode used to unlock the Safe and confirm transactions.
+                            <rect key="frame" x="0.0" y="38.5" width="374" height="81.5"/>
```
Comment (liliya-soroka) [inline Multisig/UI/Safe Management/Add Safe/CreatePasscodeSuggestionViewController.xib:44]: should not it be Safe{Wallet} mobile app ?

Comment (elgatovital) [reply Multisig/UI/Safe Management/Add Safe/CreatePasscodeSuggestionViewController.xib:44]: it is Safe{Wallet}

File: Multisig/UI/App/Onboarding/OnboardingViewController.swift (hunk 1 above)
Comment (elgatovital) [reply Multisig/UI/App/Onboarding/OnboardingViewController.swift:54]: image

looks fine

File: Multisig/UI/Settings/SafeSettingsViewController/SafeSettingsViewController.swift [hunk 3]
```diff
@@ -207,15 +207,15 @@ class SafeSettingsViewController: LoadableViewController, UITableViewDelegate, U
...
+            (section: .name("Safe Account Name"), items: [Section.Name.name(safe.name ?? "Safe \(safe.addressValue.ellipsized())")]),
 
             (section: .requiredConfirmations("Required confirmations"),
              items: [Section.RequiredConfirmations.confirmations("\(threshold) out of \(ownersInfo.count)")]),
 
             (section: .ownerAddresses("Owners"),
              items: ownersInfo.map { Section.OwnerAddresses.ownerInfo($0) }),
 
-            (section: .safeVersion("Safe version"),
+            (section: .safeVersion("Safe Account version"),
```
Comment (liliya-soroka) [inline Multisig/UI/Settings/SafeSettingsViewController/SafeSettingsViewController.swift]: On Android "Safe Account Base contract version" was used as a mastercopy version field name . Please. use the same names on both platforms

Comment (elgatovital) [reply Multisig/UI/Settings/SafeSettingsViewController/SafeSettingsViewController.swift]: changed to "Safe Account base contract version"

**Review by MouazAlzahabi — APPROVED** [review]

**Pull request merged**